controllers/admin/AdminSaferPayOfficialPaymentController.php

```diff
@@ -69,8 +70,13 @@ public function postProcess()
         /** @var SaferPayRestrictionCreator $restrictionCreator */
         $restrictionCreator = $this->module->getModuleContainer()->get(SaferPayRestrictionCreator::class);
 
+        $paymentMethodsFromSaferPay = $this->getPaymentMethods();
```

Comment:
Wouldn't use "SaferPay" in the variable name, only paymentMethods should be enough.